### ISSUE TITLE
VPN: DNS names for nodes

### ISSUE THREAD

**Issue by dsmiley** [kontena/kontena#1297]
It would be _very_ useful to me if Kontena registered DNS names for the nodes, thereby allowing me to VPN and then access the hosts over SSH.  For example `ssh core@instancename.node.kontena.local` or perhaps just `instancename.local`.  Presently I must remember the IP address.

The situation this would be _very_ useful for is those people like me that don't have public IP addresses for all nodes; they are behind NAT except one -- where I put the VPN of course.

**Comment by dsmiley** [comment]
For anyone else reading, the IP address to the nodes over VPN is of the form 10.81.0.x  where 'x' is the node number.

**Comment by SpComb** [comment]
Yes, the nodes use statically allocated weave overlay addresses of the form `10.81.0.X`.

Each node will also register itself into the DNS under the "etc" name, so you can also use that: ` getent hosts etcd.kontena.local`
```
10.81.0.4       etcd.kontena.local
10.81.0.5       etcd.kontena.local
10.81.0.3       etcd.kontena.local
```
